protocols.md

```diff
@@ -1,3 +1,101 @@
 ### Protocols
 
-*(These are mostly out of scope, but we need to describe at least one protocol to show how the layer above DIDComm builds on DIDComm constructs. This section should be short.)*
\ No newline at end of file
+*(These are mostly out of scope, but we need to describe at least one protocol to show how the layer above DIDComm builds on DIDComm constructs. This section should be short.)*
+
+
+
+#### Protocol Identifier URI
+
+A Protocol Type URI identifies protocol versions unambiguously.  Additionally, Protocol Type URIs may be used by a developer to locate documentation about a protocol. 
+
+The URI must be composed as follows:
+```
+[doc-uri][delim][protocol-name]/[semver]
+```
+With ABNF:
+```ABNF
+protocol-identifier-uri = doc-uri delim protocol-name "/" semver
+delim                   = "?" / "/" / "&" / ":" / ";" / "="
+```
+
+
+Its loose matcher regex is:
+
+    (.*?)([a-z0-9._-]+)/(\d[^/]*)/?$
+
+
+Example Protocol Type URIs:
+
+```
+http://example.com/protocols?which=lets_do_lunch/1.0
+http://example.com/protocols/lets_do_lunch/1.0
+https://github.com/hyperledger/aries-toolbox/tree/master/docs/admin-invitations/0.1
+```
+
+The goals of this URI are, in descending priority:
+
+* Code can use the URI to route messages to protocol
+  handlers using [semver rules](semver.md).
+
+* The definition of a protocol should be tied to the URI such
+  that it is semantically stable. This means that once version 1.0
+  of a protocol is defined, its definition [should not change in
+  ways that would break implementations](semver.md).
+
+* Developers can discover information about novel protocols, using
+  the URI to browse or search the web.
+
+The `doc-uri` portion is any URI that exposes documentation about
+protocols. A developer should be able to browse to that URI and use human intelligence
+to look up the named and versioned protocol.
+
+
+#### Message Type URI
+
+A __Message Type URI__ identifies message types unambiguously.
+Standardizing its format is important because it is parsed by agents that
+will map messages to handlers--basically, code will look at this string and
+say, "Do I have something that can handle this message type inside protocol
+*X* version *Y*?" When that analysis happens, it must do more than compare
```

Comment:
Should be converted to normative language.

Reply:
These comments should be made in #44, as this is a dependant PR. Sorry for the confusion